src/components/ContactsList/index.ts

```diff
@@ -24,7 +24,7 @@ export default ContactsList;
 
 // --- Utils ---
 
-async function renderContacts(filter?: string) {
+ export async function renderContacts(filter?: string) {
```

Comment:
try to follow our conventions. we already have an example of how to refresh the contactslist and how to invoke this function from outside, i mean, from other components. so, you didn't have to export this function